rust/private/repository_utils.bzl

```diff
@@ -730,3 +748,44 @@ def _get_tool_extension(ctx):
         return ".tar.xz"
     else:
         return ""
+
+def select_rust_version(versions):
+    """Select the highest priorty version for a list of Rust versions
+
+    Priority order: `stable > nightly > beta`
+
+    Note that duplicate channels are unexpected in `versions`.
+
+    Args:
+        versions (list): A list of Rust versions. E.g. [`1.66.0, `nightly/2022-12-15`]
```

Comment:
nit: missing backtick between the `0` and the `,`
```
E.g. [`1.66.0`
```

Reply:
Thanks! Fixed!